packages/contract-wrappers/CHANGELOG.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "version": "4.1.0",
+        "changes": [
+            {
+                "note":
+                    "Add a `nonce` field for `TxOpts` so that it's now possible to re-broadcast stuck transactions with higher gas cost",
```

Comment:
```suggestion
                    "Add a `nonce` field for `TxOpts` so that it's now possible to re-broadcast stuck transactions with a higher gas amount",
```